libbeat/common/config.go

```diff
@@ -190,6 +202,39 @@ func (c *Config) SetChild(name string, idx int, value *Config) error {
 	return c.access().SetChild(name, idx, value.access(), configOpts...)
 }
 
+func (c *Config) IsDict() bool {
+	return c.access().IsDict()
+}
+
+func (c *Config) IsArray() bool {
+	return c.access().IsArray()
+}
+
+func (c *Config) DebugString() string {
```

Comment:
`String` is already used as Accessor. Plus I like `DebugString()`, so it's more clear this should be used for debugging only (I hope) :) .

Not sure I like this in `go-ucfg`. Plus `go-ucfg` is meant to be 'wrapped/extended' on purpose (e.g. for dealing with vendoring/version conflicts).